Non-Standard-Operations/Computer-Room-Shutdown/index.rst

```diff
@@ -63,4 +70,56 @@ auxtel-dc01.cp.lsst.org - can always be shut down first, it is only used for ima
 
 auxtel-archiver.cp.lsst.org - can always be shut down if we are not taking data, will lose ability to ingest data in Butler if taking data
 
-auxtel-daq-mgt.cp.lsst.org - can always be shut down if we are not taking data, will lose connection/monitoring of the WREB a
\ No newline at end of file
+auxtel-daq-mgt.cp.lsst.org - can always be shut down if we are not taking data, will lose connection/monitoring of the WREB a
+
+
+Tier 2
+^^^^^^
+The following is the list computers will be powered off alerting the system owners, but not taking down the control system
+
+* Control System
+yagan[13-20].cp.lsst.org
+azar[02-03].cp.lsst.org
```

Comment:
I think we now have control system dependencies to the /net/obs nfs mount on azar02 via the scriptqueue and scheduler. If so, we should move azar02 to Tier 3 (but not azar03). @tribeiro can you confirm that is the case?

Reply:
I think at this point we can consider that we are running the system in degraded mode and won't be able to update the obs env. 

Alternatively, we can move this functionality to azar1. I don't think it is worth keeping an additional node for this.